src/main/java/com/google/javascript/clutz/AliasMapBuilder.java

```diff
@@ -0,0 +1,88 @@
+package com.google.javascript.clutz;
+
+import com.google.javascript.rhino.Node;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Closure doesn't handle alias reexports like the following well:
+ *
+ * `goog.module('bare.reexport');
+ * const {Class} = goog.require('original.module');
+ * exports = Class;`

```

Comment:
What happens in the following scenario:

```
const {Class} = goog.require('original.module');
const {Class2} = goog.require('original.module2');

exports = Class;
```

Reply:
There are 4 mappings in the map, but only one will be used, since only Class is used as an export.